components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -108,51 +75,51 @@ export default class SystemUsersDropdown extends React.PureComponent {
         };
     }
 
-    handleMakeActive = (e) => {
+    handleMakeActive = (e: {preventDefault: () => void}) => {
```

Comment:
Can we use the `Event` type here and all the other cases? We don't need to re-define `preventDefault`